dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -16,7 +16,10 @@
 
 package com.google.cloud.tools.opensource.dashboard;
 
-import freemarker.core.ParseException;
+import com.google.cloud.tools.opensource.dependencies.Artifacts;
```

Comment:
This looks like internal import order. I don't think our externally published Eclipse stylesheet matches this yet. If so, I'd prefer not to have to manually reorder our imports.

Reply:
I used the google-java-format plugin in IntelliJ to produce this order (so it's easily reproducible exclusively in OSS). I can revert this reordering if it disrupts your workflow though.

Reply:
We may want to take an action item or even an OKR to update the corresponding Eclipse plugin, but until that's done please revert.